Update active-directory-configurable-token-lifetimes.md

lousimonetti opened the pull request:
Updating Microsoft Graph References to the current Microsoft Graph Documentation.

azurecla commented:
Hi __@tsubasaxZZZ__, I'm your friendly neighborhood Azure Pull Request Bot (You can call me AZPRBOT). Thanks for your contribution!
    <span>
        This seems like a small (but important) contribution, so no contribution license agreement is required at this point. Real humans will now evaluate your PR.
    </span>

TTYL, AZPRBOT;

lousimonetti committed:
Update active-directory-configurable-token-lifetimes.md

Updating Microsoft Graph References to the current Microsoft Graph Documentation.
